opengrok-web/src/main/webapp/xref.jspf

```diff
@@ -0,0 +1,193 @@
+<%--
+$Id$
```

Comment:
Just curious, what does this actually do? Is it necessary or some leftover? Some `.jspf` files in OpenGrok have it and some don't.

Reply:
That must somehow come from history. The `$Id$` is RCS (CVS) keyword that gets expanded into file metadata (https://web.mit.edu/gnu/doc/html/cvs_17.html).